articles/postgresql/flexible-server/how-to-restore-server-cli.md

```diff
@@ -58,7 +58,8 @@ az postgres flexible-server restore --restore-time
 ```
 
 **Example:**
-Restore a server from this ```2021-03-03T13:10:00Z``` backup snapshot.
+Restore a server from this ```2021-03-03T13:10:00Z``` backup snapshot (ISO8601 format).
+`restore-time` is an optional parameter, az cli will generate this to current date time.
```

Comment:
How about this?

`restore-time` is an optional parameter which, when not provided, az cli will generate this to current date time.